app/tests/repositories/test_answer_repository.py

```diff
@@ -0,0 +1,130 @@
+# @author Lorenzo Varese
+# @created 2023-06-30
+
+import unittest
+import uuid
+from unittest.mock import MagicMock
+
+from app.stuquiz.entities.answer import Answer
+from app.stuquiz.repositories.answer_repository import AnswerRepository
+from app.tests.repositories.test_constants import TEST_ANSWER
+
+
+class TestAnswerRepository(unittest.TestCase):
+    TEST_ID = TEST_ANSWER.id
+    test_answer = TEST_ANSWER
+
+    def setUp(self) -> None:
+        self.cursor = MagicMock()
+        self.db = MagicMock()
+        self.db_provider = MagicMock()
+
+        self.db.cursor.return_value = self.cursor
+        self.db_provider.get_db.return_value = self.db
+        self.repository = AnswerRepository(self.db_provider)
+
+    def tearDown(self) -> None:
+        self.cursor = None
+        self.db = None
+        self.db_provider = None
+        self.repository = None
+
+    def test_create_answer(self):
+        answer = Answer(
+            id=self.TEST_ID,
+            question_id='question_id_1',
+            answer='Sample answer',
+            creation_date='2023-06-30',
+            correct=True,
+            points=10
```

Comment:
`points` should be a float number, not an integer.
```suggestion
            points=10.0
```